File: Aula4/ComputationalThinkingAula4/src/exercicio2.java
import java.util.Scanner;

public class exercicio2 {

	public static void main(String[] args) {
		//Entrada: O usuário digitará 3 números inteiros (n1, n2, n3)
		//Processamento: Somar os três valores obtidos e o resultado desta soma, multiplicar por ele mesmo (n1+n2+n3 = n4^2 = n5)
		//Saída: Mostrar o valor quadrado da soma dos valores (n5).
		
		int n1, n2, n3,soma, resultado;
		// int n4, n5
		
		Scanner leitor = new Scanner (System.in);
		
			System.out.println("Digite três números inteiros");
			n1 = leitor.nextInt();
			
			n2 = leitor.nextInt();
			
			n3 = leitor.nextInt();
			
			leitor.close();
			
			soma = n1+n2+n3;
			resultado = soma*soma;
			
			//n1+n2+n3 = n4^2 = n5;
			//n4 = n1 + n2 + n3;
			//n5 = n4 * n4;
			
			System.out.println("O resultado do seu cálculo é " +resultado);

	}

}

File: aula5e6/exerciciosPython/exercicio2.py
''' O usuário informa três números
inteiros, o programa soma esses três valores e depois mostra o
quadrado do resultado obtido.'''

n1 = int(input("Digite o primeiro número: "))
n2 = int(input("Digite o segundo número: "))
n3 = int(input("Digite o terceiro número: "))
soma = n1+n2+n3
quadrado = soma*soma

print("A soma dos números é {} e o quadrado da soma é {}".format(soma, quadrado)) #impressão do resultado
#print("o quadrado do número", numero, "é:", quadrado)